reference/partials/_tigris-storage-create.erb

```diff
@@ -0,0 +1,36 @@
+## Create and manage a Tigris storage bucket
+
+Creating and managing storage buckets happens exclusively via the [Fly CLI](/docs/flyctl/install/). Install it, then [sign up for a Fly account](/docs/getting-started/sign-up-sign-in/).
```

Comment:
I don't think this sentence fits here in the context of Do More. 

You could fix this by excluding the title and the first sentence from the partial and then adding a custom intro to the object storage python page.